test/helper.rb

```diff
@@ -1,4 +1,15 @@
+require 'vcr'
+
 if ENV['TRAVIS']
   require 'coveralls'
   Coveralls.wear!
 end
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'test/cassettes'
+  config.hook_into :webmock
+  config.filter_sensitive_data('<AUTH_KEY>') { ENV['RENTLINX_AUTH_KEY'] }
```

Comment:
Yes, great point. We have to be very meticulous with every cassette, before we `push` to ensure we don't leak any _new_ sensitive data also.

Betamax provides a way to look at the data of the response when doing this filters. So we could match on '{"AUTH_KEY": "SDFLKJSDF"}' and use that to dynamically strip the `"SDFLKJSDF"` part without knowing it up front. Then we could add a similar filter for the outgoing authentication header to dynamically match on that.

Another problem here is that if any of the regular content contains something that matches our username or password, it'll be replaced with '<USERNAME>' and '<PASSWORD>' respectively. That could make it easy to infer those values so we again need to be careful, and could make these ignore rules more precise.